backend/apps/owasp/models/project_health_metrics.py

```
@@ -156,6 +161,163 @@ def bulk_save(metrics: list, fields: list | None = None) -> None:  # type: ignor
+        pdf.drawCentredString(300, 700, f"Health Metrics Report for {metrics.project.name}")
+        pdf.drawCentredString(300, 680, f"Health Score: {metrics.score:.2f}")
+        table_data = [
+            ["Metric", "Value"],
+            ["Project Age", f"{metrics.age_days}/{metrics.age_days_requirement} days"],
+            [
+                "Last Commit",
+                f"{metrics.last_commit_days}/{metrics.last_commit_days_requirement} days",
+            ],
+            [
+                "Last Pull Request",
+                f"{metrics.last_pull_request_days}/{metrics.last_pull_request_days_requirement} days",
```

Ruff (0.12.2)

Line too long (102 > 99)

(E501)